src/generators/index.mjs

```diff
@@ -3,9 +3,11 @@
 import jsonSimple from './json-simple/index.mjs';
 import legacyHtml from './legacy-html/index.mjs';
 import legacyHtmlAll from './legacy-html-all/index.mjs';
+import mandoc from './mandoc/index.mjs';
 
 export default {
   'json-simple': jsonSimple,
   'legacy-html': legacyHtml,
   'legacy-html-all': legacyHtmlAll,
+  mandoc: mandoc,
```

Comment:
nit:

```suggestion
  'man-doc: mandoc,
```

Reply:
I bet prettier and eslint will do away with the inverted commas

Reply:
Technically, the name is "man-page", so I've renamed it to that (See <https://en.wikipedia.org/wiki/Man_page>)